benches/howzit/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "howzit"
```

Comment:
Just how many diffs with howzit are we going to have, and which one will be the one to review and merge? Let's settle on one version and get it to main, so that any other PR can be rebased to only have the relevant changes.